airbyte-webapp/src/views/Connector/ConnectorCard/ConnectorCard.tsx

```diff
@@ -129,7 +131,7 @@ export const ConnectorCard: React.FC<ConnectorCardCreateProps | ConnectorCardEdi
   const testConnectorWithTracking = async (connectorCardValues?: ConnectorCardValues) => {
     trackTestConnectorStarted(selectedConnectorDefinition);
     try {
-      await testConnector(connectorCardValues);
+      return await testConnector(connectorCardValues);
       trackTestConnectorSuccess(selectedConnectorDefinition);
```

Comment:
If we `return` here on line 134, doesn't that mean we will never execute the next line to track the test success?

It feels like we should instead be saving the result of `testConnector()` to a variable, then calling track, then returning that variable

Reply:
Good catch, no idea how I missed this. @timroes is there a reason not to fail the build on unreachable code?